3-platform/order/domain/src/main/java/com/_6core/platform/orderdomain/model/Order.java

```diff
@@ -0,0 +1,15 @@
+package com._6core.platform.orderdomain.model;
+
+import java.math.BigInteger;
+import java.util.Set;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Table(name = "orders")
```

Comment:
Missed @Entity annotation?

Reply:
@Entity - package persistent available when hibernate included in dependencies

Reply:
okay

Reply:
@sokimaaa, I add 'hibernate-reactive-core' dependency for all this orm-relationship

Reply:
add hibernate-reactive-core dependency for orm relationships